fdbserver/storageserver.actor.cpp

```diff
@@ -1437,7 +1437,7 @@ ACTOR Future<Void> getShardStateQ(StorageServer* data, GetShardStateRequest req)
 void merge(Arena& arena,
```

Comment:
Everything in `merge()` is unrelated to this PR.

Reply:
Let's move to #4704.